src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Basics_Shapes_Tests.cs

```diff
@@ -153,8 +154,8 @@ public void ValidateShape(string shapeName, PixelTolerance? tolerance = null)
 						}
 
 						using (var actual = (Bitmap)testResult)
-						{
-							var scale = 2.0;
+						{ 
+							var scale = AppInitializer.GetLocalPlatform() == Platform.Android ? 1.0 : 2.0;
```

Comment:
You should not hard-code the scale... since it depends on the emulator/device you're using.